.github/workflows/commit-queue.yml

```diff
@@ -21,7 +21,18 @@ jobs:
     if: github.repository == 'nodejs/node'
     runs-on: ubuntu-latest
     steps:
+      - name: Get Pull Requests
+        continue_on_error: true
```

Comment:
The problem is that the job will succeed even if there is a real error in this step.

(btw `continue-on-error`)